addons/website_sale/tests/test_website_sequence.py

```diff
@@ -14,7 +14,23 @@ def setUp(self):
         # if stock is installed we can't archive since there is orderpoints
         if hasattr(self.env['product.product'], 'orderpoint_ids'):
             product_templates.mapped('product_variant_ids.orderpoint_ids').write({'active': False})
-        product_templates.write({'active': False})
+
+        # Service on Timesheet can't be archived nor deleted (TASK ID 2221046)
+        # So if the product is service timesheet,
+        # we use the force arg to bypass the rule. Only valid in testing.
+        time_product = None
+        try:
+            time_product = self.env.ref('industry_fsm_sale.fsm_time_product')
+        except:
+            pass # It means that we are not in enterprise, the product doesn't exist
```

Comment:
This is dirty. If there is a way to check with the orm if a module is registered, this could do the trick as well and be nicer.

The simple fact to check in the community edition for an enterprise module is ugly, but a big refactor (in 13.3 or later I guess) made that everything is much better now.